sdk/communication/Azure.Communication.Rooms/src/Generated/Models/Role.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This file name should be RoleType.cs Can you please check the generated file for the latest spec